src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Image from 'next/image';
+import logo from '../../../public/img/logo.png';
+
+type logoType = {
```

Comment:
For consistency in the project, kindly use PascalCase here (uppercase `L` and uppercase `T`) `LogoType`